tests/test_uxarray_fieldset.py

```diff
@@ -115,17 +115,37 @@ def test_fesom2_square_delaunay_uniform_z_coordinate_eval():
     grid = UxGrid(ds.uxgrid, z=ds.coords["nz"], mesh="flat")
     UVW = VectorField(
         name="UVW",
-        U=Field(name="U", data=ds.U, grid=grid, interp_method=UXPiecewiseConstantFace),
-        V=Field(name="V", data=ds.V, grid=grid, interp_method=UXPiecewiseConstantFace),
-        W=Field(name="W", data=ds.W, grid=grid, interp_method=UXPiecewiseLinearNode),
+        U=Field(name="U", data=ds.U, grid=grid, interp_method=UxPiecewiseConstantFace),
+        V=Field(name="V", data=ds.V, grid=grid, interp_method=UxPiecewiseConstantFace),
+        W=Field(name="W", data=ds.W, grid=grid, interp_method=UxPiecewiseLinearNode),
     )
-    P = Field(name="p", data=ds.p, grid=grid, interp_method=UXPiecewiseLinearNode)
+    P = Field(name="p", data=ds.p, grid=grid, interp_method=UxPiecewiseLinearNode)
     fieldset = FieldSet([UVW, P, UVW.U, UVW.V, UVW.W])
 
-    assert fieldset.U.eval(time=[0], z=[1.0], y=[30.0], x=[30.0], applyConversion=False) == 1.0
-    assert fieldset.V.eval(time=[0], z=[1.0], y=[30.0], x=[30.0], applyConversion=False) == 1.0
-    assert fieldset.W.eval(time=[0], z=[1.0], y=[30.0], x=[30.0], applyConversion=False) == 0.0
-    assert fieldset.p.eval(time=[0], z=[1.0], y=[30.0], x=[30.0], applyConversion=False) == 1.0
+    assert np.isclose(
+        fieldset.U.eval(time=ds.time[0].values, z=[1.0], y=[30.0], x=[30.0], applyConversion=False),
```

Comment:
Now that time is a float (as of #2405),  this needs to change


```suggestion
        fieldset.U.eval(time=[0.], z=[1.0], y=[30.0], x=[30.0], applyConversion=False),
```